website/schema/traceabilityAggregationEvent.v0.3.0.JSONLDContext.jsonld

```diff
@@ -0,0 +1,65 @@
+{
+    "@context": {
+        "parentEPC": {
+            "@context": {
+                "itemID": "xsd:string",
+                "name": "xsd:string"
+            },
+            "@id": "xsd:string"
+        },
+        "childEPCs": {
+            "@context": {
+                "itemID": "xsd:string",
+                "name": "xsd:string"
+            },
+            "@id": "xsd:string",
+            "@container": "@list"
+        },
+        "childQuantityList": {
+            "@context": {
+                "epcClass": "xsd:string",
+                "quantity": "xsd:decimal",
+                "uom": "xsd:string"
+            },
+            "@id": "xsd:string",
+            "@container": "@list"
+        },
+        "eventID": "xsd:string",
+        "eventTime": "xsd:string",
+        "action": "xsd:string",
+        "disposition": "xsd:string",
+        "bizStep": "xsd:string",
+        "bizLocation": "xsd:string",
+        "sensorElementList": {
```

Comment:
hey @zachzeus so yeah this is where i was looking, if i have an assertion about a mine location  at a date from an independent auditor, is there any way to tie it in here?  I want to somehow include whether 3rd party auditors were given access to this mine, i want some way to be able to express that in the traceability vocab if possible

Reply:
because what really matters, is if independent 3rd parties had access to observe